src/deb/control/postinst

```diff
@@ -1,17 +1,87 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+#set -x
 set -o errexit
 
-echo 'postinst script executing'
+. /usr/share/debconf/confmodule
+
+NAME=ned
+
+TOMCAT7_USER=$NAME
+TOMCAT7_GROUP=$NAME
+HOME=/opt/plos/$NAME
+
+function process_env_template {
+	CONTEXT_TEMPALTE=$1
+
+  ls -lh $CONTEXT_TEMPALTE
+
+	echo "Processing template $CONTEXT_TEMPALTE"
+
+  eval "cat <<EOF
+$(<$CONTEXT_TEMPALTE)
+EOF
+" > $CONTEXT_TEMPALTE
+}
+
+case "$1" in
+    configure)
```

Comment:
interesting. I was following the tomcat7 deb and got this formatting. but I'll use dh_make